framework/components/ecl-date-blocks/ecl-date-blocks.scss

```diff
@@ -0,0 +1,90 @@
+/*
+ * Date block
+ * @define date-block
+ */
+
+.ecl-date-block {
+  background-color: map-get($ecl-colors, 'grey-5');
+  border-bottom: 0.2em solid map-get($ecl-colors, 'yellow-100');
+  display: inline-flex;
+  margin: 0;
+  position: relative;
+
+  &::before {
+    border-left: 1.2em solid transparent;
+    border-top: 1.2em solid #fff;
+    content: '';
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
+}
+
+.ecl-date-block__body {
+  align-items: center;
+  display: inline-flex;
+  flex-direction: column;
+  justify-content: center;
+  min-height: map-get($ecl-spacing, 'xxxl');
```

Comment:
Height and width are about sizing, not spacing. You can use some kind of "magic" numbers if necessary